src/Symfony/Component/Intl/Countries.php

```diff
@@ -38,6 +38,11 @@ public static function getCountryCodes(): array
         return self::readEntry(['Regions'], 'meta');
     }
 
+    public static function getAlpha3Codes(): array
```

Comment:
we should keep the same API as languages IMHO

https://github.com/symfony/symfony/blob/10a05c13f74408ec454e71c293ff3f8ef99a58d7/src/Symfony/Component/Intl/Languages.php#L73

but perhaps add both methods on both sides ...

For the form integration this would allow to map the choice value (alpha2) to its alpha3 code, while keeping the translated name (choice label) given we obtain the list from `getNames()`

Reply:
also we should make the data explict with a doc, like languages does

https://github.com/symfony/symfony/blob/10a05c13f74408ec454e71c293ff3f8ef99a58d7/src/Symfony/Component/Intl/Languages.php#L69

Reply:
Done